src/include/catalog/schema.h

```diff
@@ -154,7 +176,9 @@ class Schema {
     uint16_t max_varlen_size_;
     bool nullable_;
     col_oid_t oid_;
-    common::ManagedPointer<const parser::AbstractExpression> default_value_;
+
+    // TODO(John) this should go back to being a raw pointer once #386 is in
```

Comment:
Same issue with an outdated PR reference.

Reply:
Changed to reference the new issue for smart pointers (#489).